sample.env

```diff
@@ -7,3 +7,5 @@ export STRIPE_SECRET_KEY=sk_test_HbsdaR1Bn5I84vdezKa9VcvA
 export PAYPAL_USER=receive-donation_api1.test.com
 export PAYPAL_PWD=783PM8L5UQFWCK87
 export PAYPAL_SIGNATURE=AKTEYhvFqmOFYnWv4QPso.a.lYY6AMXKUhTbIGIwV-CvfMijuU.qhstP
+export OPENEXCHANGERATES_KEY=
+#Free Forever key available here https://openexchangerates.org/signup
```

Comment:
Is there anything secret about the key? Can we check it into sample.env like we do with our other test keys?